.gitignore

```diff
@@ -1,4 +1,3 @@
-.tmp
```

Comment:
Wouldn't it be easier to just ignore `.tmp` altogether and create it in the Makefile when needed?